uiowa/uiowa
# Update find text

# How to test



```
ddev composer install
```

Sync a site and uli
See that the Find Text link is still in its usual spot with icon and all
Open Find Text link and see there are new "Workflow," "Status," and "Content Type" options
Run some searches, see that things work as expected
